data/rdm/manufacturer_pids.proto

```diff
@@ -8182,5 +8182,85 @@ manufacturer {
     }
     get_sub_device_range: ROOT_DEVICE
   }
+  pid {
+    name: "PIXEL_TYPE"
+    value: 32773
```

Comment:
What are PIDs 0x8003 and 4 used for? Can you add them all to https://wiki.openlighting.org/index.php/Open_Lighting_Allocations .

We should also at least open an issue to support these in the SPI plugin (and ideally stop using personalities to change pixel types).